File: csharp/TellDontAsk/UseCase/OrderApprovalRequest.cs
using System;

namespace TellDontAsk.UseCase
{
    public class OrderApprovalRequest
    {
        private int orderId;
        private Boolean approved;

        public void setOrderId(int orderId)
        {
            this.orderId = orderId;
        }

        public int getOrderId()
        {
            return orderId;
        }

        public void setApproved(Boolean approved)
        {
            this.approved = approved;
        }

        public Boolean isApproved()
        {
            return approved;
        }
    }
}

File: src/main/java/it/gabrieletondi/telldontaskkata/useCase/OrderApprovalRequest.java
package it.gabrieletondi.telldontaskkata.useCase;

public class OrderApprovalRequest {
    private int orderId;
    private boolean approved;

    public void setOrderId(int orderId) {
        this.orderId = orderId;
    }

    public int getOrderId() {
        return orderId;
    }

    public void setApproved(boolean approved) {
        this.approved = approved;
    }

    public boolean isApproved() {
        return approved;
    }
}
